compiler/qsc_eval/src/lower.rs

```diff
@@ -153,18 +153,39 @@ impl Lowerer {
     }
 
     fn lower_callable_decl(&mut self, decl: &hir::CallableDecl) -> fir::CallableDecl {
+        fn is_instrinsic(spec_body: &SpecBody) -> bool {
+            match spec_body {
+                SpecBody::Gen(spec_gen) => matches!(spec_gen, SpecGen::Intrinsic),
+                SpecBody::Impl(_, _) => false,
+            }
+        }
+
```

Comment:
clippy is going to complain about this unused function now, so it can just be removed.
```suggestion
```